doc/tutorials/chapter_9/rohd_vf.md

```diff
@@ -0,0 +1,449 @@
+# Content
+
+- [ROHD Verification Framework](#rohd-verification-framework)
+- [Testbenches](#testbenches)
+  - [Constructing Objects](#constructing-objects)
+  - [Component's Phases](#components-phases)
+  - [Component](#component)
+  - [Stimulus](#stimulus)
+  - [Logging](#logging)
+- [Test a Counter module with ROHD-VF](#test-a-counter-module-with-rohd-vf)
+  - [A. Define Counter DUT](#a-define-counter-dut)
+  - [B. Define the Stimulus for Counter](#b-define-the-stimulus-for-counter)
+  - [C. Counter Agent](#c-counter-agent)
+  - [D. Counter Environment](#d-counter-environment)
+  - [E. Counter Test](#e-counter-test)
+  - [F. Create an Entry Point](#f-create-an-entry-point)
+
+## Learning Outcome
+
+In this chapter:
+
+- You will learn how to use ROHD-VF to perform verification on a Counter DUT. You will be constructing the DUT, Agent, Env and a Test to verify.
+
+## ROHD Verification Framework
+
+In this chapter, we are going to learn how to use ROHD-VF, a verification framework that are built upon Rapid Open Hardware Development (ROHD) framework. ROHD-VF offers testbench in a way that are similar to UVM. For those who are unfamiliar with UVM, worry not as we will covered it piece by piece as we move along with this tutorial.
+
+## Testbenches
+
+A testbench is a software used to interact with and test a device under test (DUT). ROHD-VF testbenches are organized in a modular and extenable way using simple base classes which have specific roles. Below are the diagram of the ROHD-VF testbench.
```

Comment:
"Below are the diagram of the ROHD-VF testbench." should be something like "Below is a diagram of an example ROHD-VF testbench"